app/src/main/java/com/cornellappdev/uplift/ui/screens/MainReminderScreen.kt

```diff
@@ -0,0 +1,113 @@
+package com.cornellappdev.uplift.ui.screens
+
+import androidx.compose.animation.Crossfade
```

Comment:
Android Studio is telling me that several of these imports are unused, so we could probably remove those.